src/lib.rs

```diff
@@ -1,7 +1,35 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
-    }
+#[macro_use]
+extern crate failure;
+#[macro_use]
+extern crate log;
+#[macro_use]
+extern crate proptest;
+#[macro_use]
+extern crate serde_derive;
+extern crate libtransport_http;
```

Comment:
@SamuelMarks Would it be better to rename `libtransport-http` module (see https://github.com/Fantom-foundation/libtransport-http/blob/1909df8a99894435efec2d2fcce010e7b80870d7/Cargo.toml#L2 ) into something like `fantom_transport_http` and all other crates accordingly ?

Reply:
Maybe. I think we're still deciding how generic to make each crate.